### ISSUE TITLE
[Designer][Tools- Adaptive card editor]: Cards list items present under the "Pick a sample as a starting point" dialog box are not accessible with arrow keys.

### ISSUE THREAD

**Issue by JeanRoca** [microsoft/AdaptiveCards#7690]
https://domoreexp.visualstudio.com/MSTeams/_workitems/edit/1916710

**Comment by paulcam206** [comment]
feels like a duplicate of (but definitely related to) #7699

**Comment by anna-dingler** [comment]
Closing since it is a duplicate of https://github.com/microsoft/AdaptiveCards/issues/7699
